src/effects/browse.js

```diff
@@ -12,3 +12,14 @@ export function fetchSchools() {
       querySnapshot => querySnapshot.docs.map(doc => doc.data()),
     );
 }
+
+export function fetchSubjects(termId, subjectId) {
+  return currentTermDoc
+    .collection('subjects')
+    .where('termId', '==', termId)
```

Comment:
You're not gonna need this line since this subjects subcollection is already inside the appropriate term document (`currentTermDoc`). This _does_ mean that you won't be using the argument `termId` for now but that's okay since when we implement multiple terms we will be using this.

Reply:
So if I understand your comment correctly, subjects is under currentTermDoc so .collection('subjects') is fine and the currentTerm is already matched because of db.collection('terms').doc(CURRENT_TERM). I can just remove the first .where()